packages/flutter/lib/src/semantics/semantics_service.dart

```diff
@@ -29,8 +29,12 @@ class SemanticsService {
   ///
   /// For example a camera application can use this method to make accessibility
   /// announcements regarding objects in the viewfinder.
-  static Future<void> announce(String message, TextDirection textDirection) async {
-    final AnnounceSemanticsEvent event = AnnounceSemanticsEvent(message, textDirection);
+  ///
+  /// The assertiveness level of the announcement can be determined by setting [assertivenessSetting].
+  /// Currently, this is only supported by the web engine and has no effect on other platforms.
+  /// The default mode is 'assertive' which means it will interrupt any existing annoucement.
```

Comment:
Why is this the default? I would have expected that we want to default this to polite since that's what most experts seem to recommend for announcements.

Reply:
From @yjbanov's comment on the [issue](https://github.com/flutter/flutter/issues/104109):

> If the framework-side API makes assertiveness level an optional parameter, I would also default to assertive. My reasoning is that if developer decides to make a verbal announcement, there may not be a written equivalent of it anywhere in the UI. Otherwise, they would use a live region or some other way to inform the user. If audio announcement is the only source, missing that would cause the user to lose a piece of potentially critical information.

Reply:
In one of the other comments above you quote from the official documentation:

> Normally, only aria-live="polite" is used.

It seems strange that we'd default it to the one that's less commonly used. Ideally, Flutter should do the right thing out of the box.

Reply:
My suggestion to use assertive by default is based purely on my personal logic. If the experts and the docs recommend polite as the default, let go with polite.

Reply:
Sounds good to me.